_inc/lib/core-api/wpcom-endpoints/publicize-connection-test-results.php

```diff
@@ -0,0 +1,111 @@
+<?php
+
+require_once dirname( __FILE__ ) . '/publicize-connections.php';
+
+/**
+ * Publicize: List Conection Test Result Data
+ *
+ * All the same data as the Publicize Connections Endpoint, plus test results.
+ *
+ * @since 6.8
+ */
+class WPCOM_REST_API_V2_Endpoint_List_Publicize_Connection_Test_Results extends WPCOM_REST_API_V2_Endpoint_List_Publicize_Connections {
+	public function __construct() {
+		$this->namespace = 'wpcom/v2';
+		$this->rest_base = 'publicize/connection-test-results';
+
+		add_action( 'rest_api_init', array( $this, 'register_routes' ) );
```

Comment:
I'd like to do what you suggest. Let's leave it for a later refactor, though, since doing so would require a bunch of changes on WordPress.com, too. All the endpoints there follow this pattern, and some of the loader code there expects it.